lib/Recommend.js

```diff
@@ -13,7 +13,7 @@ const panelWidth = 402;
 const panelRecHeight = 100;
 const welcomeBoxHeight = 70;
 const footerHeight = 42;
-const onboardDomain = 'https://reddit.com';
+const onboardDomain = 'https://wikipedia.com';
```

Comment:
Going to https://wikipedia.com gets me a security error because Wikipedia's SSL cert is valid for wikipedia.org but not for wikipedia.com. Should probably change this to `https://www.wikipedia.org`.